src/middlewared/middlewared/logger.py

```diff
@@ -0,0 +1,137 @@
+import os
```

Comment:
Yes, just using `import logging` would do the magic too. However my goal was avoid to change the code of service monitoring too much, that is why I preferred add the logger instead to change the code of service monitoring.